.gitignore

```diff
@@ -26,3 +26,6 @@ upstream.toml
 #### Generated Files
 components/builder-protocol/src/message
 !components/builder-protocol/src/message/mod.rs
+
+#### Vendored test helpers
+test/test_helper
```

Comment:
It seems the test.sh script for the component reaches out to this directory for some test related goodies:      `local library_install_path="$project_root/test/test_helper/$library" `

Is this dir in a fixed state that won't change?

Reply:
Yup, this dir should always be empty in the repository. We had a preference for not committing the vendored  bats helpers, and the `test/builder-graph/test.sh` script ensures that they're populated.  It does add an additional failure point in CI, but I figured that if you can't `git clone` from GitHub we're probably not going to be able to run CI anyways.